src/hooks/usePoolData.ts

```diff
@@ -33,15 +33,15 @@ export type Partners = "keep" | "sharedStake" | "alchemix"
 export interface PoolDataType {
   adminFee: BigNumber
   aParameter: BigNumber
-  apy: string // TODO: calculate
+  apy: string
```

Comment:
can we make the new things you're adding `number` or `BigNumber` instead of strings? I realize the existing type said string but that was unused and incorrect.